preprocess/code/tests/test_update_preprocess.py

```diff
@@ -0,0 +1,300 @@
+"""Unit testing for summary_stats_util using sample data"""
+import unittest
+from os.path import abspath, dirname, join
+import json
+from collections import OrderedDict
+
+TEST_DATA_DIR = join(dirname(abspath(__file__)), 'test_data')
+#INPUT_DIR = join(PREPROCESS_DIR, 'input')
+
+from msg_util import dashes, msgt, msg
+from variable_display_util import VariableDisplayUtil
+
+
+class UpdatePreprocessTest(unittest.TestCase):
+    """Unit testing class for summary_stats_util"""
+
+    def setUp(self):
+        """Load up the test file"""
+        update_01_fname = join(TEST_DATA_DIR, 'update_01.json')
+        preprocess_01_fname = join(TEST_DATA_DIR, 'preprocess_01.json')
+        update_data = open(update_01_fname, 'r').read()
+        preprocess_data = open(preprocess_01_fname, 'r').read()
+        self.update_json_01 = json.loads(update_data,
+                                         object_pairs_hook=OrderedDict)
+        self.preprocess_json_01 = json.loads(preprocess_data,
+                                             object_pairs_hook=OrderedDict)
+        self.expected_output = json.dumps({
```

Comment:
- move the `self.expected_output` content to an external file in the 'test_data' directory'
   - Load it similar to `self.update_json_01` or `self.preprocess_json_01`

Reply:
Done